low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh

```diff
@@ -272,9 +272,9 @@ fi
     # TODO: When RE brings main branch as same strucure as cortx_builds,
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
-    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/cortx-utils_last_successful/"
+    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
+    yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel
```

Comment:
this line had my review comment earlier. We are not supposed to pull in new packages, or are these already being used in cortx, we are just deploying during dev deploy ? In which case, its fine.

Reply:
Conf store cli will not work without these pkgs,